Alignment/OfflineValidation/plugins/ValidationMisalignedTracker.cc

```diff
@@ -220,15 +224,30 @@ ValidationMisalignedTracker::~ValidationMisalignedTracker() {
 //
 // member functions
 //
+void ValidationMisalignedTracker::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  std::vector<edm::InputTag> labels;
+  std::vector<std::string> associators;
+  desc.add<std::vector<edm::InputTag> >("label", labels);
+  desc.add<edm::InputTag>("label_tp_effic", edm::InputTag("FinalTrackRefitter"));
+  desc.add<edm::InputTag>("label_tp_fake", edm::InputTag("TrackRefitter"));
+  desc.add<std::vector<std::string> >("associators", associators);
```

Comment:
```suggestion
  desc.add<std::vector<edm::InputTag> >("label", {});
  desc.add<edm::InputTag>("label_tp_effic", edm::InputTag("FinalTrackRefitter"));
  desc.add<edm::InputTag>("label_tp_fake", edm::InputTag("TrackRefitter"));
  desc.add<std::vector<std::string> >("associators", {});
```

declaring the `vector`s is unnecessary